src/pages/signUp/SignUpPage.tsx

```diff
@@ -66,7 +77,13 @@ export default function SignUpPage() {
         {status === "success" && <>Success!</>}
       </MainHeadline>
       {status === "new" && !isLoggedIn && (
-        <form onSubmit={handleSubmit} ref={formRef}>
+        <Form
+          onSubmit={handleSubmit}
+          onReset={updateFormState}
```

Comment:
these are not needed :)

 onReset={updateFormState}
 onInit={updateFormState}
 onChange={updateFormState}